pkg/services/ngalert/api/authorization.go

```diff
@@ -195,7 +195,10 @@ func (api *API) authorize(method, path string) web.Handler {
 		http.MethodGet + "/api/v1/provisioning/mute-timings",
 		http.MethodGet + "/api/v1/provisioning/mute-timings/{name}",
 		http.MethodGet + "/api/v1/provisioning/alert-rules/{UID}",
-		http.MethodGet + "/api/v1/provisioning/folder/{FolderUID}/rule-groups/{Group}":
+		http.MethodGet + "/api/v1/provisioning/alert-rules/export",
+		http.MethodGet + "/api/v1/provisioning/alert-rules/{UID}/export",
+		http.MethodGet + "/api/v1/provisioning/folder/{FolderUID}/rule-groups/{Group}",
+		http.MethodGet + "/api/v1/provisioning/folder/{FolderUID}/rule-groups/{Group}/export":
```

Comment:
I think we should consider improving RBAC in provisioning because IMO these endpoints should be available to regular users, not just admins.

Reply:
Matt did you get a chance to follow up on this?

Reply:
I think it's a good idea, but should probably be done in a separate PR